checker/tests/nullness/Issue3020.java

```diff
@@ -4,6 +4,7 @@ enum Issue3020 {
     void retrieveConstant() {
         Class<?> theClass = Issue3020.class;
         // :: error: (accessing.nullable)
+        // :: error: (dereference.of.nullable)
```

Comment:
This error comes from incorrectly using `@Nullable` as the type of `theClass`. That variable should be non-null, as it is assigned a class literal. So again, we should determine proper types for things, not ignore errors.